src/storage/statistics_cl.c

```diff
@@ -530,8 +524,6 @@ stats_ndv_dump (const char *class_name_p, FILE * file_p)
 {
   MOP class_mop;
   int error = NO_ERROR;
```

Comment:
It is trivial.
Not used. But if it is used later, keep it,

Reply:
I fixed it.